app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -186,10 +140,31 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
                 displayDeleteBookmarkDialog()
                 true
             }
+            R.id.save_bookmark_button -> {
+                askToSave()
+                true
+            }
+
             else -> super.onOptionsItemSelected(item)
         }
     }
 
+    private fun askToSave() {
+        context?.let {activity ->
+            AlertDialog.Builder(activity).apply {
```

Comment:
We don't need this alert dialog. We can just save + navigate when the user presses the save button

Reply:
Sorry for the confusion I remember I told you to look at `delete_browsing_data` for the progress bar. But for destructive actions (delete) we use dialogs, but since this is a positive action (save) we don't need a dialog